addon/helpers/download.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from "luxon";
+
+export function isDownloadUrlExpired(url) {
+  const expiryRegex = /expires=([0-9]+)/;
+  const match = url.match(expiryRegex);
+  return DateTime.fromSeconds(Number(match[1])) < DateTime.now();
+}
```

Comment:
isn't the `helpers` folder usually reserved for template helpers? maybe use "utils" instead?

Reply:
Done moved utils into utils

Reply:
Done